### ISSUE TITLE
Span Multiple Paragraphs with Tag Blocks

### ISSUE THREAD

**Issue by tohuw** [Shopify/liquid#912]
How do you create liquid tag blocks that can span multiple paragraphs of text? Whenever I try I receive errors like:

    "Liquid Exception: Liquid syntax error (line 11): 'foo' tag was never closed in /some/path"

**Comment by dylanahsmith** [comment]
If you have a `foo` block tag, then you end it with an `endfoo` tag, similar to the if tag.  That error means it didn't see that block `endfoo` block delimiter.

**Comment by pushrax** [comment]
Do you mean like:

```
{% foo %}
a
b
c
{% endfoo %}
```

or 

```
{% foo
bar
baz %}
{% endfoo %}
```

(both should work)

**Comment by tohuw** [comment]
@dylanahsmith That's not my issue. I have the end tag.

@pushrax I mean like:

{% foo %}
a

b

c
{% endfoo %}

I found I can do this by adding an arbitrary whitespace to each empty newline, but I'd rather not.

**Comment by pushrax** [comment]
That's very strange. It works by default:

```
irb(main):001:0> puts Liquid::Template.parse("{% if true %}a\n\nb\nc\n{% endif %}").render
a

b
c
=> nil
irb(main):002:0> 
```

**Comment by tohuw** [comment]
@pushrax I wonder if it's some poor implementation on my part then. I originally thought I was working against something immutable within Liquid's definitions, then I started thinking maybe it was Jekyll's section schema, now I'm thinking it's probably all my fault. :)

I've modified octopress-pullquote to make this, and I can confirm the original has this same defect, which I thought was universal, but maybe not...

[Here's the relevant code](https://gitlab.com/tohuw/jekyll-pullquote/blob/d62b073ec8c2d82fbe196d4f2a5eabb8ffe67f1d/lib/jekyll-pullquote.rb). Perhaps something will jump out to you as far as what I've botched here. Many thanks!
